core/src/main/java/io/debezium/connector/cassandra/CassandraConnectorConfig.java

```diff
@@ -277,6 +305,16 @@ public int getCommitLogMarkedCompletePollInterval() {
 
     protected static final int DEFAULT_SNAPSHOT_FETCH_SIZE = 0;
 
+    /**
+     * Must be one of 'DOUBLE', 'PRECISE', or 'STRING'. The default decimal handling mode is 'DOUBLE'.
+     * See {@link DecimalHandlingMode for details}.
+     */
+    public static final String DEFAULT_DECIMAL_HANDLING_MODE = "DOUBLE";
+    public static final Field DECIMAL_HANDLING_MODE = Field.create("decimal.handling.mode")
```

Comment:
Could you please take a look at `io.debezium.relational.RelationalDatabaseConnectorConfig.DECIMAL_HANDLING_MODE` and define the `DecimalHandlingMode` to implement `EnumeratedValue`?
We might even pull-up `DecimalHandlingMode` class to `CommonConnectorConfig` and re-use it.

Reply:
I've pushed commit https://github.com/debezium/debezium-connector-cassandra/pull/78/commits/129f0283c012ba2f5654ad6476511ee20198a6dd to make this change. It's mostly a wholesale copy of `io.debezium.relational.RelationalDatabaseConnectorConfig.DECIMAL_HANDLING_MODE` with the notable change that `io.debezium.connector.cassandra.CassandraConnectorConfig.DecimalHandlingMode.asDecimalMode()` defaults to `DOUBLE` rather than `PRECISE`:

```
        public DecimalMode asDecimalMode() {
            switch (this) {
                case PRECISE:
                    return DecimalMode.PRECISE;
                case STRING:
                    return DecimalMode.STRING;
                case DOUBLE:
                default:
                    return DecimalMode.DOUBLE;
            }
        }
```

If `DecimalHandlingMode` class were eventually to be moved to `CommonConnectorConfig` then I believe the Cassandra connector could use that, even though the default decimal modes across connectors are different. The Cassandra config property is always set with a default mode when none is explicitly specified.